vcpkg_overlays/pcre2/fix-cmake.patch

```diff
@@ -0,0 +1,334 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This entire folder can be removed as I pre-emptively added pcre2 10.42#1 to the vcpkg submodule.

Reply:
Removed, thanks!